cluster/gce/gci/master.yaml

```diff
@@ -74,6 +74,33 @@ write_files:
       [Install]
       WantedBy=kubernetes.target
 
+  - path: /etc/systemd/system/kube-logrotate.timer
+    permissions: 0644
+    owner: root
+    content: |
+      [Unit]
+      Description=Hourly kube-logrotate invocation
+
+      [Timer]
+      OnCalendar=hourly
+
+      [Install]
+      WantedBy=kubernetes.target
+
+  - path: /etc/systemd/system/kube-logrotate.service
```

Comment:
Each execution of this job is "oneshot" job. But it gets run automatically several time whenever kube-logrotate.timer is fired. See https://www.freedesktop.org/software/systemd/man/systemd.timer.html.